benchmarks/benchmark.py

```diff
@@ -0,0 +1,68 @@
+'''
```

Comment:
Use double rather than single quotes. I usually only use single quotes for characters (i.e., strings of length 1).